WcaOnRails/app/models/competition.rb

```diff
@@ -233,6 +234,7 @@ def update_foreign_keys
       CompetitionDelegate,
       CompetitionOrganizer,
       DelegateReport,
+      CompetitionTab,
     ].each do |model|
       foreign_key = model.column_names.include?("competitionId") ? "competitionId" : "competition_id"
       model.where(foreign_key => id_was).update_all(foreign_key => id)
```

Comment:
PR #381 will remove the need for the updating of ids. @timhabermaas do you think this PR could go live soon?